Swiftfin/Views/SettingsView/UserDashboardView/UsersView/Components/UsersRow.swift

```diff
@@ -0,0 +1,162 @@
+                    .placeholder { _ in
+                        SystemImageContentView(systemName: "person.fill", ratio: 0.5)
+                    }
+                    .failure {
+                        SystemImageContentView(systemName: "person.fill", ratio: 0.5)
+                    }
+                    .grayscale(userActive ? 0.0 : 1.0)
+            }
+            .squarePosterStyle()
+            .posterShadow()
+            .frame(width: 60, height: 60)
+        }
+
+        // MARK: - Row Content
+
+        @ViewBuilder
+        private var rowContent: some View {
+            HStack {
+                VStack(alignment: .leading) {
+
+                    Text(user.name ?? L10n.unknown)
+                        .font(.headline)
+                        .lineLimit(2)
+                        .multilineTextAlignment(.leading)
+
+                    TextPairView(
+                        L10n.active,
+                        value: {
+                            Text(userActive ? L10n.yes : L10n.no)
+                        }()
+                    )
+
+                    TextPairView(
+                        L10n.role,
+                        value: {
+                            if let isAdministrator = user.policy?.isAdministrator {
+                                Text(isAdministrator ? L10n.administrator : L10n.user)
+                            } else {
+                                Text(L10n.unknown)
+                            }
+                        }()
+                    )
+
+                    if let lastActivityDate = user.lastActivityDate {
+                        let timeInterval = currentDate.timeIntervalSince(lastActivityDate)
+                        let twentyFourHours: TimeInterval = 24 * 60 * 60
+
+                        TextPairView(
+                            L10n.lastSeen,
+                            value: timeInterval <= twentyFourHours ?
+                                Text(lastActivityDate, format: .relative(presentation: .numeric, unitsStyle: .narrow)) :
+                                Text(lastActivityDate, style: .date)
+                        )
+                        .id(currentDate)
+                        .monospacedDigit()
+                    } else {
+                        TextPairView(
+                            L10n.lastSeen,
+                            value: Text(L10n.never)
+                        )
+                    }
+                }
+                .font(.subheadline)
+                .foregroundStyle(labelForegroundStyle, .secondary)
+            }
+        }
+
+        // MARK: - Body
+
+        var body: some View {
+            ListRow(insets: .init(horizontal: EdgeInsets.edgePadding)) {
+                userImage
+            } content: {
+                rowContent
+                    .padding(.vertical, 8)
+            }
+            .onSelect(perform: onSelect)
+            // TODO: Do we even want to allow User Deletion?
```

Comment:
I'm happy to add this but I'm kind of stuck again for the UI. I could do a delete & a multi-select for deletion but I also have a branch where I am working on allow user creation via an add button in the top left.

Would we want to have edit and add?

Reply:
I think I've got this! Let me know if this UI is bad but I think I've got a good starting point for this:
<img width="340" alt="Screenshot 2024-10-27 at 14 07 22" src="https://github.com/user-attachments/assets/0bbe0d2f-662d-4a74-b942-a1f08dd96cbb">
<img width="335" alt="Screenshot 2024-10-27 at 14 07 31" src="https://github.com/user-attachments/assets/043ca4c1-cde4-4ea6-bdaf-0025fe8176f1">

Reply:
I stole the Filters & Add button formatting from the SeeAllButton. I tried to get fancy with the icon on the outside but that might not be desirable.

As for the other select all/multi-select this is a pure drop in steal from how the DeviceView did this.

If we like this layout, after this PR I can add the filter option to the Devices as well. I also need to move Devices to be an array instead of using a box so I can do both of those at the same time. I don't even know what an add button would be on Devices so I think I would just add the filter and nothing else. For devices, I was thinking filter on Client?

Reply:
See: https://github.com/jellyfin/Swiftfin/pull/1287#issuecomment-2445014476